bess-upf/templates/sriov-device-plugin.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+{{- if .Values.config.upf.sriov.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sriov-device-plugin
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-sriov-device-plugin-amd64
+  labels:
+    tier: node
+    app: sriovdp
+spec:
+  selector:
+    matchLabels:
+      name: sriov-device-plugin
+  template:
+    metadata:
+      labels:
+        name: sriov-device-plugin
+        tier: node
+        app: sriovdp
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      initContainers:
+        - name: init-sriov-plugin
+          image: docker.io/omecproject/omec-cni:1.1.6
+          imagePullPolicy: IfNotPresent
+          command: ["bash", "-c"]
+          args:
+            - cp /tmp/cni/bin/{sriov,vfioveth,jq,static,dhcp} /host/opt/cni/bin/;
+              chmod +x /host/opt/cni/bin/vfioveth
+          volumeMounts:
+            - name: cni-bin
+              mountPath: /host/opt/cni/bin
+      serviceAccountName: sriov-device-plugin
+      containers:
+        - name: kube-sriovdp
+          image: docker.io/nfvpe/sriov-device-plugin:v3.3
```

Comment:
Is there a new version for this image?

Reply:
Let me check and if there is new version, lets take it as part of separate PR as mentioned above.